src/SDKs/TrafficManager/Management.TrafficManager/Microsoft.Azure.Management.TrafficManager.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.TrafficManager</PackageId>
     <Description>Microsoft Azure Management TrafficManager Library</Description>
     <AssemblyName>Microsoft.Azure.Management.TrafficManager</AssemblyName>
-    <VersionPrefix>2.3.0-preview</VersionPrefix>
+    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
@vladca you can add text in <PackageReleaseNotes> tag in csproj under <PackageTags>